azure-pipelines.yml

```diff
@@ -113,6 +113,57 @@ jobs:
         ]
       ServiceEndpointUrl: 'https://api.esrp.microsoft.com/api/v1'
 
+  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
```

Comment:
Can we use just one ESRP task?

Reply:
haven't found a decent way to do that (yet)

Reply:
yes, just updated the code